_security-analytics/usage/findings.md

```diff
@@ -37,12 +37,31 @@ As one more alternative, you can select an option from the **Recently used date
 
 ## The Findings list
 
-The Findings list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.
+The **Findings** list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.
 <br><img src="{{site.url}}{{site.baseurl}}/images/Security/finding-list.png" alt="A list of all findings" width="85%">
+
 Use the **Rule severity** dropdown list to filter the list of findings by severity. Use the **log type** dropdown list to filter the list by log type.
-Each finding in the list includes a finding ID. You can select the ID to open the Finding details pane, which describes the finding by parameters defined when creating the detector.
-The Actions column includes two options for each finding:
-* The diagonal arrow provides another way to open the Findings detail pane.
-* The bell icon allows you to open the Create detector alert trigger pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
+
+The **Actions** column includes two options for each finding:
+* The diagonal arrow provides a way to open the **Finding details** pane, which describes the finding by parameters defined when creating the detector and includes the document that generated the finding.
```

Comment:
Is it always one document, or can it be multiple documents? In the UI, it says "Documents".

Reply:
In process. Thanks.